### PR TITLE
Trunk/docs

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

**Review by BobDotCom — CHANGES_REQUESTED** [review]
I made some suggestions, but I haven't finished all the files yet

File: .gitignore [hunk 1]
```diff
@@ -21,3 +20,5 @@ __pycache__
 .vs/slnx.sqlite
 env/
 build/
+node_modules/*
+test.py
```
Comment (Lulalaby) [inline .gitignore:24]:
```suggestion
test.py
*.png
```

File: .gitignore [hunk 2]
```diff
@@ -10,7 +10,6 @@ docs/crowdin.py
 *.mp3
 *.m4a
 *.wav
-*.png
```
[the remark this answers is not in the document]
Comment (VincentRPS) [reply .gitignore:13]: when i was uploading the newer images i had to remove png's from the ignore because the new images would've been ignored

**Review by BruceCodesGithub — DISMISSED** [review]
Seems good, a lot of grammar fixed

File: examples/app_commands/info.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import discord
+from discord.ext import commands
+# imports
+
+intents = discord.Intents(
+            guilds=True,
```
Comment (Middledot) [inline examples/app_commands/info.py:6]: What’s with the indents here?

Comment (Lulalaby) [reply examples/app_commands/info.py:6]: idk

File: README.rst [hunk 4]
```diff
@@ -131,7 +130,7 @@ Note: Make sure you do not reveal your bot token to anyone, it can grant access
 Links
 -----
 
-- `Documentation <https://docs.pycord.dev/en/master/index.html>`_
+- `Documentation <https://docs.pycord.dev/en/latest/index.html>`_
```
Comment (Lulalaby) [inline README.rst:133]:
```suggestion
- `Documentation <https://docs.pycord.dev/en/master/index.html>`_
```

**Review by Middledot — DISMISSED** [review]

**Review by Lulalaby — COMMENTED** [review]
LGTM for me

**Review by VincentRPS — APPROVED** [review]

File: discord/commands/core.py [hunk 5]
```diff
@@ -793,6 +793,129 @@ def _update_copy(self, kwargs: Dict[str, Any]):
...
+            "private": ChannelType.private_thread,
+            "news": ChannelType.news_thread,
+        }
+        self._type = type_map[thread_type]
+    
+    @property
+    def __name__(self):
+        return 'ThreadOption'
+
+class Option:
```
Comment (VincentRPS) [inline discord/commands/core.py:817]: @Lulalaby something broke here

Comment (Lulalaby) [reply discord/commands/core.py:817]: Meh fix it

**Review by VincentRPS — APPROVED** [review]

**Review by Middledot — DISMISSED** [review]

**Review by Middledot — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**